src/actions/ast.js

```diff
@@ -19,7 +19,7 @@ import {
   getFramework
 } from "../workers/parser";
 
-import type { SourceId } from "debugger-html";
+import type { SourceId } from "../types";
```

Comment:
This is kind of annoying have multiple `../types` and `./types` imports. In perf.html we have a single `src/types/` folder where we collect pure-type files. This has been a nice practice for us.